### PR TITLE
Update disclaimer card component for reuse 

### PR DESCRIPTION
**Pull request by sven1103**
- Provides a simple API to enable/disable confirmation features
- Adds a configurable title
- Left-aligns the content to harmonise the layout with the overall alignment strategy

### CODE REVIEW

File: vaadinfrontend/src/main/java/life/qbic/datamanager/views/general/DisclaimerCard.java [hunk 1]
```diff
@@ -13,10 +13,16 @@
  * <b>Disclaimer Card</b>
  *
  * <p>Disclaimer cards are special cards that render a disclaimer to the user,
- * with a clickable button for confirmation.</p>
+ * with an clickable confirmation element to either confirm or redirect the user to a follow up
```
Comment (wow-such-code) [inline vaadinfrontend/src/main/java/life/qbic/datamanager/views/general/DisclaimerCard.java:16]:
```suggestion
 * with a clickable confirmation element to either confirm or redirect the user to a follow up
```
🧐

File: vaadinfrontend/src/main/java/life/qbic/datamanager/views/general/DisclaimerCard.java [hunk 2]
```diff
@@ -13,10 +13,16 @@
...
+ * step.</p>
  * <p>
  * The component fires a {@link DisclaimerConfirmedEvent} after a user has confirmed the
  * disclaimer.
+ * <p>
+ * The card consists of a title that can be set, as well as a disclaimer text and an clickable
```
Comment (wow-such-code) [inline vaadinfrontend/src/main/java/life/qbic/datamanager/views/general/DisclaimerCard.java:22]:
```suggestion
 * The card consists of a title that can be set, as well as a disclaimer text and a clickable
```

File: vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/experiments/experiment/ExperimentDetailsComponent.java [hunk 3]
```diff
@@ -98,7 +98,7 @@ public ExperimentDetailsComponent(@Autowired ProjectInformationService projectIn
   }
 
   public DisclaimerCard createDisclaimer() {
-    var disclaimer = DisclaimerCard.create("No experiment variables defined", "Add");
+    var disclaimer = DisclaimerCard.create("No experiment variables defined", "Add", "Missing variables");
```
Comment (wow-such-code) [inline vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/experiments/experiment/ExperimentDetailsComponent.java:101]: NIT: Three string parameters in a weird order (title last) could get confusing. I think a factory would be overkill, though?

Comment (sven1103) [reply vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/experiments/experiment/ExperimentDetailsComponent.java:101]: alternatively I can create more explicit creation methods. BTW: these are factory creation methods :D

Comment (wow-such-code) [reply vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/experiments/experiment/ExperimentDetailsComponent.java:101]: okay, fair enough :)

**Review by wow-such-code — DISMISSED** [review]

**Review by wow-such-code — APPROVED** [review]
that's nicer, thanks!

**Pull request merged**